__tests__/app.test.js

```diff
@@ -90,6 +90,40 @@ describe('GET /api/articles:article_id', () => {
 //     });
 // });
 
+
+describe('GET /api/articles/:article_id/comments', () => {
+    test('STATUS 200: reponds with an array of comments for the given article_id', () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(result => {
+            const {comments} = result.body
+            console.log(comments)
+            expect(comments.length).toBe(11)
+            for (const comment of comments) expect(comment.article_id).toBe(1)
```

Comment:
We should be checking the shape of the returned data objects here - making sure all the required properties are present